toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -154,34 +169,60 @@ func updateGrubConfigForVerity(rootfsVerity imagecustomizerapi.Verity, rootHash
 	return nil
 }
 
-func constructVerityKernelCmdlineArgs(rootfsVerity imagecustomizerapi.Verity, rootHash string,
+func constructVerityKernelCmdlineArgs(verityInfo map[string]map[string]string,
 	partIdToPartUuid map[string]string, partitions []diskutils.PartitionInfo, buildDir string,
 ) ([]string, error) {
-	// Format the dataPartitionId and hashPartitionId using the helper function.
-	formattedDataPartition, err := systemdFormatPartitionId(rootfsVerity.DataDeviceId,
-		rootfsVerity.DataDeviceMountIdType, partIdToPartUuid, partitions, buildDir)
-	if err != nil {
-		return nil, err
-	}
+	var newArgs []string
+	addedVerityFlag := false
```

Comment:
I don't think the `constructVerityKernelCmdlineArgs` function is called unless there is at least one verity device. So, `newArgs ` can probably be initialized with the `rd.systemd.verity=1` arg.